### ISSUE TITLE
Heading level

### ISSUE THREAD

**Issue by jdahm** [rlister/org-present#8]
Thanks for contributing this! I've used it for a few times now to present material at meetings, and really like it.

I'm not sure how easy this is, but at least once so far I would have liked to have the slides be defined via the level 2 heading and have a single line at the top like

* Level_1_Heading ** Level_2_Heading
... Content ...

Looking at the code it seems like this could be fairly simple, since there are places where the level is hard-coded, and could instead be a variable. Then we only need to take care of the header line, which would involve a bit more effort. If I have some time this weekend I might take a look.

**Comment by rlister** [comment]
@jdahm I agree this is a good idea. If you have a go at it send me a pull request. Otherwise, I'll take a look if I get some time this week.

**Comment by Kazark** [comment]
Another thing to consider here would be, since we are coming from an Org tree, rather than thinking of these presentations through the very flat format of traditional presentations, to adapt them more to the natural structure of org trees. Something akin to narrowing and widening (though having the nested headers display in that case might be a good idea). In reveal.js they also have some non-linear concept of slides (it's two-dimensional, I think). Perhaps we could have here the idea of not just "next slide" and previous slide, but "next slide, same header level", "next slide, deeper level" etc. I think that would make the presentations most natural emerge from the concepts of Org mode.
